integration-testing/src/functionalTest/kotlin/kotlinx.atomicfu.gradle.plugin.test/framework/runner/Utils.kt

```diff
@@ -24,11 +28,56 @@ internal fun GradleBuild.buildGradleByShell(
     return BuildResult(exitCode, logFile)
 }
 
+internal fun Path.enableCacheRedirector() {
+
+    val cacheRedirectorPath =
+        System.getProperty("cache.redirector.path").ifBlank { null } ?: error("Cache redirector has not been set")
+
+    // Path relative to the current gradle module project dir
+    val redirectorScript =
+        Paths.get(cacheRedirectorPath)
+            .toAbsolutePath()
+            .normalize()
+            .toFile()
+
+    val gradleDir = resolve("gradle").also { it.createDirectories() }
```

Comment:
```suggestion
    val gradleDir = resolve("gradle").createDirectories()
```

nitpick: `createDirectories()` returns the same value, so `also {}` could be removed.

Reply:
moved this change to https://github.com/Kotlin/kotlinx-atomicfu/pull/460